sk0-back/src/auth/auth.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+import { JwtModule } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+import { AuthService } from 'src/users/services/auth.service';
+import { JwtStrategy } from './jwt.strategy';
+
+@Module({
+  imports: [
+    PassportModule,
+    JwtModule.register({
+      secret: process.env.JWT_SECRET,
```

Comment:
Please use the centralized configuration instead of `process.env` there should be an async injection with the configService. If you are blocked by this please ping me.

Reply:
Yeah I did that but I didn't commit because it won't compile